src/config/src/meta/stream.rs

```diff
@@ -706,7 +780,7 @@ pub struct StreamSettings {
     #[serde(skip_serializing_if = "Option::None")]
     pub flatten_level: Option<i64>,
     #[serde(skip_serializing_if = "Vec::is_empty", default)]
-    pub defined_schema_fields: Vec<String>,
+    pub defined_schema_fields: Vec<DataField>,
```

Comment:
The UDS don't care about type. it only care about field name.

Reply:
maybe we shouldn't save datatype in UDS, because the schema will change, the datatype will change, if we save the type in UDS, we also need to update UDS when schema update. @oasisk